tools/hooks/lint.rb

```diff
@@ -47,6 +47,8 @@ def run_haml(files)
 
 def lint_failure(output)
   puts output
+  `which terminal-notifier`
+  `terminal-notifier -message "Lint failed"` if $?.success?
```

Comment:
I think this could be `if system('which terminal-notifier')`
http://ruby-doc.org/core-2.2.0/Kernel.html#method-i-system